src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -90,6 +96,8 @@ public void requestWithoutHeaderPassed() throws Exception {
 
     /**
      * LastModifiedCachingWire can cache GET requests.
+     * @todo: #120 inline redundant "headers" variable
+     *  see http://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html
```

Comment:
@westonized generally these puzzles are to trivial, it's 5 min in IntelliJ, not enough for full 30 min task. Can we join them together?